### PR TITLE
[ADD] add sale visible tax, that show the subtotal tax and ...

### PR DESCRIPTION
**Pull request by sebastienbeau**
... subtatotal tax inc on sale order line and invoice line

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Fields should be defined as readonly. The rest LGTM.

**Comment by sebastienbeau** [comment]
I do not think we need to add the readonly here, indeed the two fields are function field without inverse function so there are in readonly (as we can write on it). You can try it (you will see that the field are in readonly in the interface)

**Comment by pedrobaeza** [comment]
Cool, I didn't know that. I have tried and it works perfectly.

Thanks.

:+1:

File: sale_visible_tax/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#
+###############################################################################
+
+{'name': 'Sale Visible Tax',
+ 'version': '0.0.1',
+ 'author': 'Akretion',
+ 'website': 'www.akretion.com',
+ 'license': 'AGPL-3',
+ 'category': 'Generic Modules',
+ 'description': """This module add the field "Sub total tax inc"
```
Comment (eLBati) [inline sale_visible_tax/__openerp__.py]: add/adds
field/fields

**Comment by eLBati** [comment]
:+1:

File: sale_visible_tax/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ """,
+ 'depends': [
+     'sale',
+ ],
+ 'data': [
+     'sale_view.xml',
+     'invoice_view.xml',
+ ],
+ 'installable': True,
+ 'application': True,
```
Comment (joaoalf) [inline sale_visible_tax/__openerp__.py]: I'm not sure that this module qualifies as application.

**Comment by joaoalf** [comment]
It's not a show stopper but have you considered splitting away the invoice part into another module (account_invoice_visible_tax?)?

**Comment by foutoucour** [comment]
Except the orthography in description pointed out by @eLBati, :+1:

**Comment by sebastienbeau** [comment]
Hi I did the change. it should be ok for merging.
Regarding the split of the module, it can be a possibity. But as I never had a customer case that do accounting without sale (I think that the majority of customer use both), I am not sure a such litlle module need to be splitted. But if one someone have the use case and need to show the taxe only on invoice, I will be totaly agree to split it.
Thanks

**Pull request merged**